docs/README.md

```diff
@@ -1,6 +1,6 @@
 # Repository for code and instructions for building OpenJDK
```

Comment:
Not sure about moving this file to docs. I'd prefer it to stay in the root of the repo